package-lock.json

```diff
@@ -2427,6 +2436,15 @@
         "follow-redirects": "^1.14.8"
       }
     },
+    "azure-openai": {
```

Comment:
Why do we need to install the dependency but ended up not using it? Am I missing something? or perhaps I need an explanation of how we are going to use it.

Reply:
Sorry about this; I have removed it and pushed a new commit.